### PR TITLE
Realtime check for available shortlink

### PR DESCRIPTION
**Pull request by tdtgit**
* Enable shortlink auto check when users type to `.custom-url-field`
* Remove `.check-link-availability` button

polr

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -11,32 +11,36 @@ $(function() {
...
-            } else if (msg == 'available') {
-                $('#link-availability-status').html('<span style="color:green"><i class="fa fa-check"></i> Available</span>');
-            } else if (msg == 'invalid') {
-                $('#link-availability-status').html('<span style="color:orange"><i class="fa fa-exclamation-triangle"></i> Invalid Custom URL Ending</span>');
-            } else {
-                $('#link-availability-status').html(' <span style="color:red"><i class="fa fa-exclamation-circle"></i> An error occured. Try again</span>' + msg);
-            }
-        });
+    var timeoutId;
+    $('.custom-url-field').keyup(function (){
```
Comment (overint) [inline public/js/index.js]: I think we should wait until the user has finished typing before checking if it is available.
I've used the [typewatch jquery](https://github.com/dennyferra/TypeWatch) plugin before for this purpose.

Comment (tdtgit) [reply public/js/index.js]: Okay I will update this PR soon

Comment (overint) [reply public/js/index.js]: No worries.

Comment (tdtgit) [reply public/js/index.js]: Updated @overint

**Comment by overint** [comment]
Any thoughts @cydrobolt?

**Pull request closed without merging**